PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingServiceEventHandlers.cs

```diff
@@ -116,14 +126,31 @@ private void Debugger_DebuggerStop(object sender, DebuggerStopEventArgs e)
                         file = _mapRemoteToLocal[bp.Script];
                     }
 
-                    _callback.DebuggerStopped(new DebuggerStoppedEventArgs(file, bp.Line, bp.Column));
+                    _callback.DebuggerStopped(new DebuggerStoppedEventArgs(true, file, bp.Line, bp.Column, false));
                 }
             }
             else
-            {
+            {                
                 if (_callback != null)
                 {
-                    _callback.DebuggerStopped(new DebuggerStoppedEventArgs());
+                    if (_attaching)

```

Comment:
Only other attribute I can think of off the top of my head would the name of the runspace which is usually "RemoteRunspace" or something like that, but I feel like that is even more fragile. We can't go off of the type of connection either since the NamedPipeConnection type is not a public type. Hence having the local process attaching case be the last case in GetCallStack. (PowerShellDebuggingService.cs apx line 900)